src/redux/topic/saga.js

```diff
@@ -223,51 +222,21 @@ export function* calculateQtumWinningsRequestHandler() {
         headers: { 'Content-Type': 'application/json' },
       };
 
-      const result = yield call(request, `${bodhiapi}/qtumwinnings`, options);
-
-      const value = result ? convertBNHexStrToQtum(result['0']) : undefined;
-
-      yield put({
-        type: actions.CALCULATE_QTUM_WINNINGS_RETURN,
-        value,
-      });
-    } catch (error) {
-      yield put({
-        type: actions.CALCULATE_QTUM_WINNINGS_RETURN,
-        value: { error: error.message ? error.message : '' },
-      });
-    }
-  });
-}
-
-export function* calculateBotWinningsRequestHandler() {
-  yield takeEvery(actions.CALCULATE_BOT_WINNINGS, function* onCalculateBotWinningsRequest(action) {
-    const {
-      contractAddress,
-      senderAddress,
-    } = action.payload;
-
-    try {
-      const options = {
-        method: 'POST',
-        body: JSON.stringify({
-          contractAddress,
-          senderAddress,
-        }),
-        headers: { 'Content-Type': 'application/json' },
-      };
-
-      const result = yield call(request, `${bodhiapi}/botwinnings`, options);
-
-      const value = result ? convertBNHexStrToQtum(result['0']) : undefined;
+      const result = yield call(request, Routes.winnings, options);
+      let value;
+      if (result) {
+        const botWon = convertBNHexStrToQtum(result['0']);
+        const qtumWon = convertBNHexStrToQtum(result['1']);
+        value = [botWon, qtumWon];
```

Comment:
You are doing action.qtumWon and action.botWon in reducer. Should this be value ={
botWon,
qtumWon
}
?

Reply:
Sure. Changed to use an object instead. Can you test with yours? For the completed events, none of mine have any winning amounts > 0.

Reply:
Ah wait haha, none of the existing contracts on chain will be able to return anything since they dont have the `calculateWinnings` method. Will just return 0, 0.

Gonna deploy some more beta contracts for next round of beta testing.

Reply:
Yeah, sure we can test when it's switched. Just need to remember this change.